### ISSUE TITLE
I wonder what is the mean of global(type_of_learning_rate)?

### ISSUE THREAD

**Issue by yimjinkyu1** [zadaianchuk/HyperGradientDescent#2]
I test your tensorflow code about hypergradient descet.
But "Adam Train loss graph" is very fluctuated.
I have some of question.
First, Adam_HD_optimizer don't have mu2 about v(Adam have two momentum hyperparameter)
Second, What is the mean of global type?
Is it hd ? not global means native adam algorithm?
